DEB-42 Update README

de-laz opened the pull request.

de-laz committed:
Update readme

This commit will add a summary of the patient registration/update
worflow to the README of the repository.

Related Tickets:
 - DEB-3
 - DEB-42

bradsawadye commented on README.md:
```diff
@@ -15,6 +15,16 @@ The following channels are set up:
 - PUT  <http://localhost:5001/patient-registration/{FHIR_ID}>
 - POST <http://localhost:5001/immunization>
 
+### Patient Registration/Update flow
```
@de-laz it looks good man. I think we should move the section up so that it comes before the examples and also change the heading to h2. We should also add a link to the docasaurus page

de-laz replied:
@bradsawadye It sounds like a good idea. Thanks :+1:

de-laz committed:
Move the patient registration/update workflow

This commit will move the "Patient Registration/Update flow" section
up to a higher place on the README file. It makes sense to have this
section much higher on the document as it summuerizes one of the
workflows being implemented by this package.

https://github.com/jembi/covid19-immunization-tracking-package/pull/6#discussion_r611419323

Related Tickets
 - DEB-41
 - DEB-3

de-laz committed:
Add link to Instant OpenHIE docs

Related Tickets:
 - DEB-41
 - DEB-3

de-laz committed:
Add link to OpenHIM Mapping Mediator site

Related Tickets:
 - DEB-41
 - DEB-3
